api/v1beta1/gitrepository_types.go

```diff
@@ -71,6 +69,7 @@ type GitRepositorySpec struct {
 // GitRepositoryRef defines the git ref used for pull and checkout operations.
 type GitRepositoryRef struct {
 	// The git branch to checkout, defaults to master.
+	// +kubebuilder:default:=master
```

Comment:
Should this be changed to main to align with the toolkit?

Reply:
I am personally in favour of this change, as I think we should abandon non-inclusive terms as much as we can, even if this means users need to take an extra step because they have not made the change themselves yet. Now would also be the right moment. as we already introduce a set of breaking changes in the upcoming release.

@stefanprodan wdyt?

Reply:
This will break our e2e tests, docs and guides since podinfo is using `master`. If I change podinfo's default branch to `main` then I'll break Flux v1 and Flagger docs, workshops, blog posts, etc...

Reply:
Ok lets skip this for now as the consequences are a bit too large. We can track this in a separate issue, and we can deal with it when other things have aligned to use main in the future.